src/test/predict_validation_test.cpp

```diff
@@ -165,6 +165,32 @@ TEST_F(PredictValidation, RequestWrongBatchSizeAuto) {
     EXPECT_EQ(status, ovms::StatusCode::BATCHSIZE_CHANGE_REQUIRED);
 }
 
+TEST_F(PredictValidation, RequestWrongBatchSizeAutoBinaryInputs) {
+    modelConfig.setBatchingParams("auto");
+    tensorflow::serving::PredictRequest binaryInputRequest = preparePredictRequest(
+        {
+            {"Binary_Input",
+                std::tuple<ovms::shape_t, tensorflow::DataType>{{1, 224, 224, 3}, tensorflow::DataType::DT_STRING}},
```

Comment:
This is most likely wrong, shape of binary input has 1 dimension but you specify 4